apps/api/src/app/workflows-v2/dtos/step-issues.dto.ts

```diff
@@ -17,7 +18,7 @@ export class StepIssuesDto {
   })
   @IsOptional()
   @ValidateNested()
-  controls?: Record<string, StepContentIssueDto[]>;
+  controls?: Record<string, RuntimeIssue[]>;
```

Comment:
created the `RuntimeIssue` class that is reused for both steps and layouts

Reply:
Awesome!